gtk4/Cargo.toml

```diff
@@ -37,9 +37,11 @@ optional = true
 git = "https://github.com/gtk-rs/lgpl-docs"
 
 [dependencies]
+async-trait = "0.1"
```

Comment:
Let's do this manually for now instead of adding another dependency. It's easy enough

Reply:
Done